rust/sbp/src/sbp2json.rs

```diff
@@ -161,7 +163,7 @@ fn add_common_fields<'a>(
     }
 
     let sender_id = msg.get_sender_id();
-    let size = msg.sbp_size();
+    let size = payload[MSG_HEADER_SIZE_OFFSET] as usize;
```

Comment:
Why change this?

Reply:
I should put this back, this was originally where the `String` sizing issue manifested